src/main/scala/lms/thirdparty/nccl/nccl.md

```diff
@@ -0,0 +1,12 @@
+This folder tries to support NCCL (https://github.com/NVIDIA/nccl) as thirdparty library.
```

Comment:
maybe rename this file to `README.md` or move this to the doc folder?

Reply:
did